.changeset/eleven-ways-count.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-eth": patch
+---
+
+change listr to listr2
```

Comment:
copies this change https://github.com/scaffold-eth/create-eth/pull/53/files#diff-32998dd93341cf2d3e267eb301f7e3b4d41155a54ec890935f4bc161b6700461R6

Reply:
I think this file can be removed

Reply:
Done! https://github.com/scaffold-eth/create-eth/pull/53/commits/ac1dbd9822ff59298ba86e88b0affca18d307319